integration-tests/rpc-get-protocol-constants.spec.ts

```diff
@@ -0,0 +1,528 @@
+            expect(constants.proof_of_work_threshold.toString()).toEqual("70368744177663");
+            expect(constants.tokens_per_roll.toString()).toEqual("8000000000");
+            expect(constants).toHaveProperty("michelson_maximum_type_size", 1000);
+            expect(constants.seed_nonce_revelation_tip.toString()).toEqual("125000");
+            expect(constants).toHaveProperty("origination_size", 257);
+            expect(constants.block_security_deposit.toString()).toEqual("512000000");
+            expect(constants.endorsement_security_deposit.toString()).toEqual("64000000");
+            if (constants.baking_reward_per_endorsement) {
+                expect(constants.baking_reward_per_endorsement.toString()).toEqual("1250000,187500");
+            } else {
+                expect(constants).toBeUndefined();
+            }
+            expect(constants.endorsement_reward.toString()).toEqual("1250000,833333");
+            expect(constants.cost_per_byte.toString()).toEqual("250");
+            expect(constants.hard_storage_limit_per_operation.toString()).toEqual("60000");
+
+            if (constants.test_chain_duration) {
+                expect(constants.test_chain_duration?.toString()).toEqual("1966080");
+            } else {
+                expect(constants.test_chain_duration).toBeUndefined();
+            }
+
+            if (constants.quorum_min) {
+                expect(constants).toHaveProperty("quorum_min", 2000);
+            } else {
+                expect(constants.quorum_min).toBeUndefined();
+            }
+
+            if (constants.quorum_max) {
+                expect(constants).toHaveProperty("quorum_max", 7000);
+            } else {
+                expect(constants.quorum_max).toBeUndefined();
+            }
+
+            if (constants.min_proposal_quorum) {
+                expect(constants).toHaveProperty("min_proposal_quorum", 500);
+            } else {
+                expect(constants.min_proposal_quorum).toBeUndefined();
+            }
+
+            if (constants.initial_endorsers) {
+                expect(constants).toHaveProperty("initial_endorsers", 24);
+            } else {
+                expect(constants.initial_endorsers).toBeUndefined();
+            }
+
+            if (constants.delay_per_missing_endorsement) {
+                expect(constants.delay_per_missing_endorsement.toString()).toEqual("8");
+            } else {
+                expect(constants.delay_per_missing_endorsement).toBeUndefined();
+            }
+
+        } catch (ex) {
+            fail(ex);
+        }
+        done();
+    })
+
+})
+
+describe('Fetch constants for Proto7 from delphinet', () => {
+    // delphinet is taking longer to call, so set timeout
+    jest.setTimeout(30000);
+    it('succesfully fetches constants for Proto7', async (done) => {
+        try {
+            const constants = await TezosDelphi.rpc.getConstants();
```

Comment:
Since Tezos upgraded to Delphi, this test is no longer required (the precedent test is sufficient, constants can be fetched from mainnet for proto7)

Reply:
removed this test